src/hooks/data/vaults/fallbacks/pricer/useAladdinApiPricer.ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios"
+import { Address, useQuery } from "wagmi"
```

Comment:
You need to use the react-query import. This is a custom implementation internal to wagmi and should only be used for interacting with contracts.

Reply:
done